### DISCUSSION TITLE
bitnami

### DISCUSSION THREAD

**Discussion by cihance** [PHOENIX-MEDIA/magento2-helm#8] [General]
Hallo 
thank you sharing resources and knowledge.
Bitnami is no more supported by gcp. any alternative resource / way to replace it?

**Comment by phoenix-bjoern** [comment]
@cihance can you share issue description on the Bitnami components? IMHO there shouldn't be any limitation deploying the resources if e.g. StorageClass and IngressClass are correctly configured (see https://github.com/PHOENIX-MEDIA/magento2-helm/blob/main/values_gke.yaml).

**Comment by cihance** [comment]
thank you for your reply

We have used the resources and now seeking an optimal way to reduce costs.

For that reason i wanted to ask you if you can share your experience for a cost effective infrastructure. 

Would having a single node e2-standard-2 machine be enough to sustain it? what would you suggest to have minimum and maximum number of pods..

and if it wouldnt be too much to ask, What would it be minimum and ideal RAM, CPU to assign each pods ?

**Reply by phoenix-bjoern** [reply]
@cihance I'm sorry, we won't give any cloud provider recommendation. However, if you don't have experience with Kubernetes we recommend a managed K8S offering.
Server resources depend on the load. For a simple environment you'll need 4-8 cores + 16-32 GB RAM.
Will close this issue.
